src/commands/deploy.rs

```diff
@@ -93,6 +93,10 @@ pub struct DeployCommand {
         env = DEPLOYMENT_ENV_NAME_ENV
     )]
     pub deployment_env_id: Option<String>,
+
+    /// Pass a key/value (key=value) to all components of the application.
+    #[clap(short, long, parse(try_from_str = parse_kv))]
```

Comment:
IMO `kv` is a more descriptive short flag and less likely to conflict with later flags
```suggestion
    #[clap(short = "kv", long, parse(try_from_str = parse_kv))]
```

Reply:
I believe short flags are a single character.  I agree with removing `-k` as the short flag though